docs/index.bs

```diff
@@ -1723,6 +1847,7 @@ spec: webidl; urlPrefix: https://heycam.github.io/webidl/
     <pre class="idl" id="fetch-event-init-dictionary">
       dictionary FetchEventInit : ExtendableEventInit {
         required Request request;
+        required Promise&lt;Response?&gt; preloadResponse;
```

Comment:
Again the intent is for this to be a promise that resolves with a response or undefined.